packages/web-components/src/index.html

```diff
@@ -5,10 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
     <title>Stencil Component Starter</title>
 
-    <script type="module" src="/build/component-library.esm.js"></script>
-    <script nomodule src="/build/component-library.js"></script>
     <link rel="stylesheet" href="https://unpkg.com/@department-of-veterans-affairs/formation@6.16.2/dist/formation.min.css">
     <link rel="stylesheet" href="build/component-library.css">
+    <script type="module" src="/build/component-library.esm.js"></script>
+    <script nomodule src="/build/component-library.js"></script>
```

Comment:
Oh, so we just needed to load the CSS first?

Reply:
It seems this is only needed for IE11. Selectors such as `:hover` receive the correct styling. I'm guessing this is a repaint bug with IE11?